### PR TITLE
PGIR DR1 light curves

### PR DESCRIPTION
**Pull request by jacquesalice**
Hi @bmerino95 can you please review this PGIR notebook? The pgir_dr1.photometry table is still being copied over to prod, so you'll have to set the profile to db01 for now (`qc.set_profile('db01')`). It should be complete in the next day or two, so we won't merge this PR until then

### CODE REVIEW

**Comment by kareninysimba** [comment]
I will strongly suggest to please change the nbid to something else (0069 or higher) since @rnikutta and I agreed that nbid would be reserved to the legacy survey notebook that we are working on. That is already in the name of all tables and indices used by such nb in the database and in the tap_schema.
I will highly appreciate it.
Thanks!

**Comment by rnikutta** [comment]
Yes, please bump nbid up to 0069 @jacquesalice , thanks!

**Comment by jacquesalice** [comment]
> I will strongly suggest to please change the nbid to something else (0069 or higher) since @rnikutta and I agreed that nbid would be reserved to the legacy survey notebook that we are working on. That is already in the name of all tables and indices used by such nb in the database and in the tap_schema. I will highly appreciate it. Thanks!

Done!

File: 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb [hunk 1]
```diff
@@ -0,0 +1,808 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a7db7e3-56d2-4947-bd8f-d5630d529f1b",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"goals\"></a>\n",
+    "# Goals & notebook summary\n",
+    "The goal of this notebook is demonstrate how to generate light curves from the **Palomar Gattini-IR (PGIR) Data Release 1 (DR1)** catalog of infrared J-band light curves and also preform a basic Lomb-Scargle periodogram analysis to identify a period. \n",
```
Comment (bmerino95) [inline 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb:50]: The goal of this notebook is **to** demonstrate

Comment (bmerino95) [inline 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb:50]: ... light curves and also **'perform'** a ...

File: 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb [hunk 2]
```diff
@@ -0,0 +1,808 @@
...
+    "\n",
+    "**Description of Palomar Gattini-IR:** PGIR is a wide-field, robotic, near-infrared time domain survey covering the entire visible night sky north of declination -28.5 at a median cadence of 2 nights. The survey operates in a single filter (J-band, calibrated to the 2MASS system), with a single exposure field of view of 25 square degrees and a native pixel scale of 8.7 arcsec/pixel. Further details about the instrument and data reduction system can be found in [De et al. (2020)](https://ui.adsabs.harvard.edu/abs/2020PASP..132b5001D/abstract). Light curves are extracted by performing Point Spread Function (PSF) photometry on the stacked images from each visit of a field, with the entire observing footprint divided into 1,329 fields. The first data release contains J-band light curves of approximately 286 million sources from the 2MASS catalog, with a total of approximately 50 billion photometric measurements [(Murakawa et al. 2024)](https://ui.adsabs.harvard.edu/abs/2024arXiv240601720M/abstract). \n",
```
Comment (bmerino95) [inline 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb:52]: 'Each visit of a field' can be rewritten as 'each field visit'

File: 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb [hunk 3]
```diff
@@ -0,0 +1,808 @@
...
+    "from dl import queryClient as qc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2c31c1ec-b1cf-49df-b82f-a7807fd02727",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"lightcurve\"></a>\n",
+    "# Input search coordinate and generate light curve\n",
```
Comment (bmerino95) [inline 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb:145]: I think coordinate should be plural here.

File: 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb [hunk 4]
```diff
@@ -0,0 +1,808 @@
...
+    "\n",
+    "### Enter RA and Dec (in degrees) of object and cone-search radius\n",
```
Comment (bmerino95) [inline 03_ScienceExamples/PGIRLightCurves/PGIR_DR1_light_curves.ipynb:147]: (in degrees) of **'an'** object

**Review by bmerino95 — COMMENTED** [review]
Hi @jacquesalice. I just finished running the notebook and didn't run into any issues. I caught a few spelling mistakes, but other than that, it looks good!

**Comment by jacquesalice** [comment]
Thanks @bmerino95 ! I implemented all of your suggested changes. Can you please Approve the PR? 

We'll still wait to merge until the photometry table is done copying over

**Review by bmerino95 — APPROVED** [review]
Looks good!

**Pull request merged**